features-json/css-anchor-positioning.json

```diff
@@ -708,7 +708,8 @@
   "notes":"",
   "notes_by_num":{
     "1":"Can be enabled via the `#enable-experimental-web-platform-features` flag in `chrome://flags`",
-    "2":"Can be enabled using the `layout.css.anchor-positioning.enabled` flag. Enabled by default in Firefox Nightly only"
+    "2":"Can be enabled using the `layout.css.anchor-positioning.enabled` flag. Enabled by default in Firefox Nightly only",
+    "3": "There's a [bug with `position: fixed` and non-initial containing blocks](https://codepen.io/pavelp/pen/KwVPrrB)."
```

Comment:
I'm not sure if a link to a personal codepen is appropriate here, so I'm happy to use it as a conversation starter and discuss what a more durable alternative could be if this change is overall acceptable.